contractcourt/htlc_timeout_resolver.go

```diff
@@ -462,6 +462,7 @@ func (h *htlcTimeoutResolver) sweepSecondLevelTx() error {
 			Fee: sweep.FeePreference{
 				ConfTarget: secondLevelConfTarget,
 			},
+			Force: true,
```

Comment:
🤦

Fine with getting in as is for rc3, but we should circle back here with additional test coverage.